build.sbt

```diff
@@ -1,17 +1,21 @@
 import microsites.ExtraMdFileConfig
 
+
 lazy val buildSettings = Seq(
   organization := "com.github.finagle",
   version := "0.22.0",
   scalaVersion := "2.12.6",
+  scalafixSemanticdbVersion := "4.0.0-M1",
```

Comment:
I'd assume we only needed semanicdb/scalafix for migration purposes? If so let's remove these before merging.